cmd/experimental/kjobctl/pkg/cmd/describe/describe.go

```diff
@@ -191,6 +191,11 @@ func (o *DescribeOptions) Run(ctx context.Context) error {
 			continue
 		}
 
+		labels := obj.GetLabels()
```

Comment:
can't we solve this here 

https://github.com/kubernetes-sigs/kueue/blob/ba37646431e0e3eedfad874a09692df7dc32a0e2/cmd/experimental/kjobctl/pkg/cmd/describe/describe.go#L245-L260

and avoid listing unnecessary objects?

Reply:
Unfortunately no. ResourceBuilder does not allow to filter by label selectors and name.

```
Error: name cannot be provided when a selector is specified
```